setup.py

```diff
@@ -546,7 +546,7 @@ def is_py_64():
     print("failed to read readme: ignoring...")
     readme = __doc__
 
-setup(name="psycopg2",
+setup(name="psycopg2-yb",
```

Comment:
Did we decide on `psycopg2-yugabytedb` ?

Reply:
At that time we had decided `psycopg2-yb` but I will change it to `psycopg2-yugabytedb`